src/app.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta name="description" content="Listen to podcasts on the web, wherever!" />
     <link rel="manifest" href="%svelte.assets%/manifest.json" />
+    <link rel="preconnect" href="https://sklgqhbbbhticzdgegcj.supabase.co" />
```

Comment:
issue: I don't think we should hard-code this

suggestion: in the `app` layer, there are a few components that manage meta tags, perhaps you could move it there and use the value from the environment variable?